playbooks/openstack-caasp_instance.yml

```diff
@@ -163,27 +152,60 @@
             - skip_ansible_lint
 
         - name: Extend inventory in workspace with newly created caasp nodes
-          config_template:
-            src: "{{ socok8s_workspace }}/inventory/skeleton-inventory.yml"
+          copy:
+            content: "{{ caasp_overrides }}"
             dest: "{{ socok8s_workspace }}/inventory/caasp.yml"
-            config_type: yaml
-            config_overrides: "{{ caasp_overrides }}"
           vars:
-            caasp_overrides:
+            caasp_overrides: |
               caasp-admin:
-                hosts: "{{ admininventory }}"
+                hosts:
+              {% for node in admins %}
+                  {{ node.Name }}:
+                    ansible_host: {{ node.Networks.split(',')[-1].replace(' ','') }}
+              {% endfor %}
+
               caasp-masters:
-                hosts: "{{ masterinventory }}"
+                hosts:
+              {% for node in masters %}
+                  {{ node.Name }}:
+                    ansible_host: {{ node.Networks.split(',')[-1].replace(' ','') }}
+              {% endfor %}
+
               caasp-workers:
-                hosts: "{{ workerinventory }}"
+                hosts:
+              {% for node in workers %}
+                  {{ node.Name }}:
+                    ansible_host: {{ node.Networks.split(',')[-1].replace(' ','') }}
+              {% endfor %}
+
               airship-openstack-control-workers:
-                hosts: "{{ controlinventory }}"
+                hosts: &firsttwoworkers
```

Comment:
is this correct or a typo? The others have `*` instead (which Im ignorant on what is doing to be honest)

Reply:
`&` is for storing a value, `*` is to load it.

Reply:
:+1: